server/jest-controller/list.js

```diff
@@ -29,14 +29,31 @@ module.exports = async () => {
           "interactive-flows.optional",
         ],
         SEP31: ["toml", "info"],
+        SEP6: [
+          "toml",
+          "info",
+          "sep10",
+          "deposit",
+          "withdraw",
+          "transaction",
+          "transactions",
+          "fee.optional",
+        ],
       };
 
+      let testProject = process.env.PROJECT ? process.env.PROJECT : "SEP24";
+
       const unorderedTests = output
         .trim()
         .split("\n")
         .map((line) => {
-          const testName = path.basename(line).split(".test.js")[0];
-          if (orderedTests[testProject].includes(testName)) {
+          if (path.dirname(line) == `cases-${testProject}`) {
```

Comment:
Do we need this line? I was able to remove it previously by adding line 9 above does that not work for your use case?

Reply:
Oh I think that was a merging mistake, I'll remove it